tembo-operator/src/ingress.rs

```diff
@@ -274,7 +274,7 @@ pub async fn reconcile_postgres_ing_route_tcp(
     let mut present_ing_route_tcp_names_list: Vec<String> = vec![];
 
     // Check for all existing IngressRouteTCPs in this namespace
-    // Filter out any that are not for this DB or do not have the correct prefix
+    // Filter out any that are not for this DB, do not have the correct prefix, or do not have matching service name
```

Comment:
This was originally programmed like this to aid in the migration from a different service name and domain name to new service name and domain name, but now we only need to protect from domain name swapping. The point of this is so that if we add different domain names to the data plane, it doesn't ever remove one of the domains from the ingressroutetcp, which would break existing connections, for example instances that are still connected via `...coredb.io` domain

Reply:
but I think we don't need this any more, so I just simplified the whole block